.yamllint

```diff
@@ -0,0 +1,21 @@
+extends: default
+rules:
+  line-length:
+    max: 160
+  document-start: disable
+  indentation:
+    indent-sequences: true
+
+  # Allows aligning subsequent lines with [] sequences
+  brackets:
+    min-spaces-inside: 0
+    max-spaces-inside: -1
+  commas:
+    max-spaces-after: -1
+
+  # Allows aligning key value pairs
+  colons:
+    max-spaces-after: -1
+
+  truthy:
+    check-keys: false
```

Comment:
This will allow us the use the same config in other repos, as long as only the ignored files are different:
```
ignore-from-file: [.gitignore, .yamlignore]
```

Reply:
added in [c82e558](https://github.com/os-autoinst/os-autoinst-common/commit/c82e5584184650236fea313e69ba09f4031e2476)